aperturedb/SingleEntityBlobNewestCSV.py

```diff
@@ -0,0 +1,226 @@
+from aperturedb import CSVParser
+import logging
+import hashlib  # for sha1
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+
+# SingleEntityBlobNewestCSV
+# Update an Entity which has an associated blob to the data in the CSV
+# What this means is:
+# - If it doesn't exist, add it.
+# - If it exsits and the blob hasn't changed, update it.
+# - If it exists and the blobl has changed, delete and re-add it.
+
+# This means if these elements are part of a graph where they are linked by connections
+#  these connections will be need to be regenerated afterwards.
+
+# This class utilizes 3 conditionals
+# - normal constraint_ to select the element
+# - a series of updateif_ to determine if an update is necessary
+# - one or more prop_ and the assocaited updateif blob conditonals
+#   to determine if a update or an delete/add is appropriate
+
+# Generated fields
+#  currently blobsha1 and blobsize are the generated fields
+#  which this class creates and manages for you that allow it to make the determination
+#  - blobsha1 - the sha1 for the blob is calculated
+#  - blobsize - the length in bytes of the blob is calcualte
+#
+#  the result is then used to identify if a blob has changed.
```

Comment:
needs to be a doc string.
Also there must be a snippet of the CSV with essential fields.